secant/Models/SensitiveData.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  SensitiveData.swift
+//  secantTests
+//
+//  Created by Lukáš Korba on 06.02.2023.
+//
+
+import Foundation
+import ZcashLightClientKit
+
+// MARK: - Redacted Protocol
+
+/// `Undescribable` comes from the SDK and it is a reliable and tested protocol ensuring custom
+/// destriptions and dumps never print outs the exact value but `--redacted--` instead.
+/// `Redacted` protocol is just a helper so we can let developers to see the sensitive data when
+/// developing and debugging but production or release builds (even testflight) are set to redacted by default.
+/// A special build scheme `<...>-unredacted` is set to hold `UNREDACTED` flags.
+#if UNREDACTED
+protocol Redacted { }
+#else
+protocol Redacted: Undescribable { }
+#endif
+
+// MARK: - Redacted Seed Phrase
+
+/// Read-only redacted holder for a seed phrase.
+struct SeedPhrase: Codable, Equatable, Redacted {
+    private let seedPhrase: String
+
+    init(_ seedPhrase: String) {
+        self.seedPhrase = seedPhrase
+    }
+    
+    /// This method returns seed phrase, all 24 words with no protection or support of `Redacted` protocol.
+    /// Use it wisely and never log outcome of this method or share with anybody.
+    func value() -> String {
+        seedPhrase
+    }
+}
+
+// MARK: - Redacted Birthday
+
+/// Read-only redacted holder for a birthday.
+struct Birthday: Codable, Equatable, Redacted {
+    private let birthday: BlockHeight?
+
+    init(_ birthday: BlockHeight?) {
+        self.birthday = birthday
+    }
+
+    /// This method returns birthday with no protection or support of `Redacted` protocol.
+    /// Use it wisely and never log outcome of this method or share with anybody.
+    func value() -> BlockHeight? {
+        birthday
+    }
+}
+
+// MARK: - Redacted String
+
+/// Redacted holder for a string.
+struct RedactedString: Equatable, Hashable, Redacted {
```

Comment:
I think this name best describes this type
```suggestion
struct RedactableString: Equatable, Hashable, Redacted {
```